.github/workflows/devcontainer-shellcheck.yml

```diff
@@ -0,0 +1,25 @@
+name: Devcontainer Shellcheck
+
+on:
+  pull_request:
+    paths:
+      - ".devcontainer/**/*.sh"
+  push:
+    paths:
+      - ".devcontainer/**/*.sh"
+
+permissions:
+  contents: read
+
+jobs:
+  devcontainer_shellcheck:
+    name: Devcontainer Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout (GitHub)
+        uses: actions/checkout@v4
+
+      - name: Lint Devcontainer Scripts
+        run: |
+          sudo apt install -y shellcheck
```

Comment:
It looks pre-installed on `ubuntu-latest` (aka 24.04):
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#installed-apt-packages

Reply:
Ah nice.